### ISSUE TITLE
Graph tab can be readded: getNeighborhood is fixed

### ISSUE THREAD

**Issue by audrism** [ssc-oscar/DRE#86]

**Comment by dlomax64** [comment]
getNeighbors is working now, but FastGraph is giving a typeMap[t] is undefined error. I'm not really familiar with how it works, but I can try to figure it out.

**Comment by audrism** [comment]
If you can create a PR that just re-enables fastgraph, I can look at it myself. 

It would be more important to help bootstrap the cs340 team (just sent email) first

**Comment by dlomax64** [comment]
Okay. I'm talking with James to try to figure it out, but I will go ahead and re-add.

**Comment by audrism** [comment]
btw, graph still shows nothing, but no crash

**Comment by dlomax64** [comment]
Yes, it is still giving the typeMap[t] is undefined error. I spoke with James about it and he wasn't sure what the problem was. I'll see if I can figure it out
